src/components/daily-dropoffs-report.tsx

```diff
@@ -124,7 +125,7 @@ function DailyDropffsReport(): VNode {
   useResizeListener(() => setWidth(ref.current?.offsetWidth));
   useAgencies(data);
 
-  const nonNullData = data || [];
+  const nonNullData = aggregate(data || []);
```

Comment:
Just to confirm: The other places we use `data` or call `loadData` don't need to be using the aggregated result?

https://github.com/18F/identity-reporting/blob/0afee59a170b0ac1aebe2a54c7dba21976bdfc45/src/components/daily-dropoffs-report.tsx#L126

https://github.com/18F/identity-reporting/blob/0afee59a170b0ac1aebe2a54c7dba21976bdfc45/src/components/daily-auths-report.tsx#L233-L235

Reply:
Correct, it shouldn't matter, `useAgencies` will grab the unique agencies, so it doesn't matter if it's been grouped by issuer or not (since there can be multiple issuers per agency anyways)